esy.json

```diff
@@ -32,7 +33,7 @@
   },
   "resolutions": {
     "@opam/conf-libev": "esy-packages/libev:package.json#86d244e",
-    "@opam/conf-autoconf": "esy-packages/esy-autoconf:package.json#71a8836",
+    "@opam/conf-autoconf": "esy-packages/esy-autoconf:package.json#94f846f",
```

Comment:
Thanks for sending this along. I'd love to get this through CI to test what the impact of changes are, but it's choking on this autoconf change. Is there a reason why this particular resolution is included here?

Reply:
Before the change, I got the following error:
```
echo "Updating man page config.guess.1"
Updating man page config.guess.1
PATH="../tests:../config:$PATH"; \
export PATH; \
/bin/sh /home/xxx/.esy/3_________________________________________________________________/b/opam__s__conf_autoconf-4e230a19/config/missing --run help2man \
    --include=./config.guess.x \
    --include=./common.x \
    --output=config.guess.1 config.guess
help2man: can't get `--help' info from config.guess
Try `--no-discard-stderr' if option outputs to stderr
make[1]: *** [Makefile:311: config.guess.1] Error 1
make[1]: Leaving directory '/home/xxx/.esy/3_________________________________________________________________/b/opam__s__conf_autoconf-4e230a19/man'
make: *** [Makefile:193: all-recursive] Error 1
error: command failed: 'make' (exited with 2)
esy-build-package: exiting with errors above...
```

Reply:
I reverted the change, but it's still failing to build.

Reply:
that's annoying - it's flaky for some reason. I bet if I retry it it'll work...

Reply:
@314eter If you rebase on current master this issue should go away. Thanks for your patience.